doc-Installing_on_AWS/topics/installation.adoc

```diff
@@ -181,17 +181,21 @@ To import the appliance:
 ]
 ----
 +
-. Use AWS-CLI tools to import the image. Run the following command:
+. Use AWS-CLI tools to import the disk as a snapshot by following the steps in the
+link://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-import-snapshot.html[Importing a Disk as a Snapshot Using VM Import/Export] documentation.
```

Comment:
Hi @jhoyt-rh my recommendation here is we need to remove the deep link and replace with a link to aws documentation landing page and a direction to "See the AWS documentation on importing a disk as a snapshot using VM import/Export." We'll avoid a future broken link of Amazon decides to move this particular doc elsewhere.

Reply:
Removed the deep link, by linking to the AWS documentation landing page and indicating the appropriate section with the instructions to be used.